Marketplace/repositories/carrito.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../database/models");
+const entidad = db.Carrito;
+
+module.exports = {
+	ObtenerTodos: (usuarioID) => {
+		return entidad.findAll({
+			include: ["producto", "imagen"],
+			where: { usuario_id: usuarioID },
+		});
+	},
+	EliminarRegistro: (registroID) => {
+		return entidad.destroy({
+			where: { id: registroID },
+		});
+	},
+	ActualizarCarrito: (infoCarrito, cantRegistros) => {
+		for (let i = 0; i < cantRegistros; i++) {
```

Comment:
Este for deberia estar en el controller. Por otro lado, lo que estas llamando "registro" + i, deberia ser idElemento o algo asi, para que se entienda a que se esta haciendo referiencia

Reply:
Me parece bien que el _for_ esté en el controlador. Fijate si te parece mejor cómo quedó. Gracias.